javascript/packages/orchestrator/src/spawner.ts

```diff
@@ -82,7 +82,7 @@ export const spawnNode = async (
   if (node.accounts) {
     // check if the node directory exists if not create (e.g for k8s provider)
     let nodeFilesPath = client.tmpDir;
-    if (parachain && parachain.name) nodeFilesPath += `/${parachain.name}`;
+    // if (parachain && parachain.name) nodeFilesPath += `/${parachain.name}`;
```

Comment:
Sorry I miss this change, the idea behind using the the `parachains name` is to group all the artifacts of the parachain in the same `directory`. Also, worth nothing that this is the `local` path used in the spawn phase and not the real `keystore` in the node.

Reply:
Yes. My point here is that this path `nodeFilesPath += `/${parachain.name}'` is only created for the keystores which makes us end up with:
```
----> zombie-random-dir/collator01/(all dirs and configs etc)
          zombie-random-dir/100/collator01/keystores/<the keystores only>
```
which to me makes not much of sense - as just the keystores are separated.

In addition since this is just the local path - I do not see any added value on having it separated (since we are the only ones using it - and not the node

Reply:
Hey @wirednkod, the main reason of this separation is that in the `native` provider we create a new directory for each node as isolated environment that `acts` as node root directory. So, you can think of `zombie-random-dir/collator01/(all dirs and configs etc)` as the `isolated running node`, and the other one is the directory we use to store all the created artifacts needed to run the `node`. So, in native they can look similar (and have dups files) but that is because from zombienet perspective they are different.

Make sense? Thanks!!